test_aodndata/moorings/test_abosFileClassifier.py

```diff
@@ -205,22 +205,81 @@ def test_sofs_surface_properties_dm(self):
         self.assertEqual(dest_dir, 'IMOS/ABOS/SOTS/2014')
         self.assertEqual(dest_filename, filename)
 
+    def test_sofs_waves_rt(self):
+        filename = 'IMOS_ABOS-SOTS_W_20190318_SOFS_FV00_SOFS-8-2019-MRU-Surface-wave-height-realtime.nc'
+        testfile = os.path.join(self.tempdir, filename)
+        make_test_file(testfile,
+                       {'time_coverage_start': "2019-03-18T00:00:00Z",
+                        'time_coverage_end': "2019-10-31T00:00:00Z"}
+                       )
+        dest_dir, dest_filename = os.path.split(AbosFileClassifier.dest_path(testfile))
+        self.assertEqual(dest_dir, 'IMOS/ABOS/SOTS/2019/real-time')
+        self.assertEqual(dest_filename, filename)
+
+    def test_sofs_oldnames_waves_rt(self):
+        filename = 'IMOS_ABOS-ASFS_W_20150202T000000Z_SOFS_FV01_TriAXYS.nc'
+        testfile = os.path.join(self.tempdir, filename)
+        make_test_file(testfile,
+                       {'time_coverage_start': '2015-01-01T00:00:00Z',
+                        'time_coverage_end': '2015-01-01T23:30:00Z'},
+                       HAV={},
+                       HMAX={},
+                       HSIG={},
+                       TSIG={},
+                       VDIR={}
+                       )
+        dest_dir, dest_filename = os.path.split(AbosFileClassifier.dest_path(testfile))
+        self.assertEqual(dest_dir, 'IMOS/ABOS/ASFS/SOFS/Surface_waves/Real-time/2015_daily')
+        self.assertEqual(dest_filename, filename)
+
+    @unittest.skip("not implemented yet - no files so far")
```

Comment:
What's the benefit of adding these? We can add them when there's some new behaviour we want to test.

Reply:
Benefit1: explicit is better than implicit and it is a future placeholder for me/you/whoever.
Benefit2: Follow the expectations of test order in regard to product names/types. 
Benefit3: avoid question likes why surface_properties is missing !? 
Benefit4: You can easily search/match the product name instead of a name being missing in the file.

Reply:
Ok :+1: